src/main/haskell/kore/src/Kore/Step/AxiomPatterns.hs

```diff
@@ -6,24 +6,65 @@ module Kore.Step.AxiomPatterns
     , koreIndexedModuleToAxiomPatterns
     ) where
 
+import Data.Default
+       ( Default (..) )
 import Data.Either
        ( rights )
+import Data.Functor
+       ( ($>) )
+import Data.Ord
+       ( comparing )
+import Data.Semigroup
+       ( (<>) )
 
-import Kore.AST.Common
-import Kore.AST.Kore
-import Kore.AST.MetaOrObject
-import Kore.AST.PureML
-import Kore.AST.PureToKore
-       ( patternKoreToPure )
-import Kore.AST.Sentence
-import Kore.ASTUtils.SmartPatterns
-import Kore.Error
-import Kore.IndexedModule.IndexedModule
-import Kore.Predicate.Predicate
-       ( CommonPredicate, wrapPredicate )
+import           Kore.AST.Common
+import           Kore.AST.Kore
+import           Kore.AST.MetaOrObject
+import           Kore.AST.PureML
+import           Kore.AST.PureToKore
+                 ( patternKoreToPure )
+import           Kore.AST.Sentence
+import           Kore.ASTUtils.SmartPatterns
+import           Kore.Attribute.Parser
+                 ( ParseAttributes (..) )
+import qualified Kore.Attribute.Parser as Attribute
+import           Kore.Error
+import           Kore.IndexedModule.IndexedModule
+import           Kore.Predicate.Predicate
+                 ( CommonPredicate, makeTruePredicate, wrapPredicate )
 
 newtype AxiomPatternError = AxiomPatternError ()
 
+data AxiomAttributes =
+    AxiomAttributes
+        { axiomOrdering :: !Ordering
+        }
+  deriving (Eq, Ord, Show)
+
+instance Default AxiomAttributes where
+    def = AxiomAttributes { axiomOrdering = EQ }
+
+instance ParseAttributes AxiomAttributes where
+    attributesParser =
+        do
+            axiomOrdering <-
+                Attribute.choose (Attribute.choose getHeat getCool) (pure EQ)
+            return AxiomAttributes {..}
```

Comment:
I think that this is a rather opaque way of returning things that works here because the AxiomAttributes definition fits in the same screen as this, but wouldn't work if more things are inserted between the definition and this line. Why wouldn't you explicitly mention the things that go in AxiomAttributes?

Reply:
I think you are right that `RecordWildCards` can make things like this unclear. Let's go ahead and remove all the uses and disable that extension, before we acquire more instances of it.